docs-2.0/3.ngql-guide/7.general-query-statements/6.show/17.show-sessions.md

```diff
@@ -0,0 +1,63 @@
+# SHOW SESSIONS
+
+`SHOW SESSIONS`语句显示所有会话信息，也可以指定会话ID进行查看。
+
+## 注意事项
+
+登录数据库时，会创建一个会话，操作结束执行`exit`退出登录时，会清空自身的会话信息。
+
+如果没有正常退出，且没有在配置文件[nebula-graphd.conf](../../../5.configurations-and-logs/1.configurations/3.graph-config.md)设置空闲会话超时时间（`session_idle_timeout_secs`），会话不会自动释放。
+
+对于未自动释放的会话，需要手动删除指定会话(TODO: coding)。
+
+## 语法
+
+```ngql
+SHOW SESSIONS;
+SHOW SESSION <Session_Id>;
+```
+
+## 示例
+
+```ngql
+nebula> SHOW SESSIONS;
++------------------+----------+--------------------+----------------------------+----------------------------+---------------+----------+----------------+
+| SessionId        | UserName | SpaceName          | CreateTime                 | UpdateTime                 | GraphAddr     | Timezone | ClientIp       |
++------------------+----------+--------------------+----------------------------+----------------------------+---------------+----------+----------------+
+| 1623305056644097 | "user1"  | ""                 | 2021-06-10T06:04:16.644097 | 2021-06-10T06:04:16.638039 | "graphd:9669" | 0        | "172.22.xx.xx" |
++------------------+----------+--------------------+----------------------------+----------------------------+---------------+----------+----------------+
+| 1623304491050858 | "root"   | "basketballplayer" | 2021-06-10T05:54:51.50858  | 2021-06-10T06:17:31.5417   | "graphd:9669" | 0        | "172.22.xx.xx" |
++------------------+----------+--------------------+----------------------------+----------------------------+---------------+----------+----------------+
+
+nebula> SHOW SESSION 1623304491050858;
++--------------+-----------------------------+
+| VariableName | Value                       |
++--------------+-----------------------------+
+| "SessionID"  | 1623304491050858            |
++--------------+-----------------------------+
+| "UserName"   | "root"                      |
++--------------+-----------------------------+
+| "SpaceName"  | "basketballplayer"          |
++--------------+-----------------------------+
+| "CreateTime" | 2021-06-10T05:54:51.50858   |
++--------------+-----------------------------+
+| "UpdateTime" | 2021-06-10T06:17:34.866137  |
++--------------+-----------------------------+
+| "GraphAddr"  | "graphd:9669"               |
++--------------+-----------------------------+
+| "Timezone"   | 0                           |
++--------------+-----------------------------+
+| "ClientIp"   | "172.22.xx.xx"              |
++--------------+-----------------------------+
+```
+
+|参数|说明|
+|:---|:---|
+|`SessionId`|会话ID，唯一标识一个会话。|
+|`UserName`|会话的登录用户名称。|
+|`SpaceName`|用户当前所使用的图空间。刚登录时为空（`""`）。|
+|`CreateTime`|会话的创建时间（UTC时间），即用户认证登录的时间。|
```

Comment:
it means the localtime，the timezone used by the `timezone_name` in the config file.